src/api/order.ts

```diff
@@ -0,0 +1,35 @@
+import { apiClient } from './apiClient';
+import { toast } from 'react-toastify';
+import { NavigateFunction } from 'react-router-dom';
+
+type OrderParams = {
+  productId: number;
+  sender: string;
+  recipients: {
+    name: string;
+    phone: string;
+    quantity: number;
+  }[];
+};
+
+export const fetchOrderSubmit = async (
+  order: OrderParams,
+  token: string,
+  navigate: NavigateFunction
+) => {
+  try {
+    await apiClient.post('/order', order, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    toast.success('주문이 완료되었습니다!');
+  } catch (err: any) {
```

Comment:
[ change request ] 1. any 타입 제거

any를 사용하면 Typescript를 배우는 의미가 많이 퇴색됩니다..!! Javascript랑 똑같으니까요 ㅠ

아래 내용처럼 수정해보면서 체득해보면 좋을 거 같네요..!
이 파일 외에도 몇 군데 더 있는 거 같으니 한꺼번에 수정 부탁드려요.

```suggestion
  } catch (err: unknown) {
    if (isAxiosError(err) && err.response?.status === 401) {
```

수정이 끝나시면 `npm run build & npm run lint` 돌려보시는 거 잊지 마시구요!